src/firmware/main/primitives/accurate_shoot.c

```diff
@@ -188,13 +187,28 @@ static void accurate_shoot_tick(log_record_t *log)
     }
     else
     {
+        FirmwareRobot_t* robot = app_firmware_world_getRobot(world);
+
         // accelerate at ball to kick it
-        chicker_auto_arm((global_params->extra & 1) ? CHICKER_CHIP : CHICKER_KICK,
-                         global_params->params[3]);
-        if (!(global_params->extra & 1))
+        Chicker_t* chicker = app_firmware_robot_getChicker(robot);
+        bool chipping      = global_params->extra & 1;
+        if (chipping)
+        {
+            float chip_distance = global_params->params[3];
+            app_chicker_enableAutokick(chicker, chip_distance);
+        }
+        else
         {
-            dribbler_set_speed(8000);
+            float speed_m_per_s = global_params->params[3];
+            app_chicker_enableAutochip(chicker, speed_m_per_s);
         }
+
+        if (!chipping)
+        {
+            Dribbler_t* dribbler = app_firmware_robot_getDribbler(robot);
+            app_dribbler_setSpeed(dribbler, 8000);
+        }
+
```

Comment:
Can't comment directly but line `318` has `apply_accel` without going through the robot. May want to comment out the function to help find all usages.

Reply:
_hilarious_ fact: this function was already deleted. C just implicitly declares it (see our issue with `limit` the other day)

changed to use `app_control_applyAccel`

Reply:
oh yikes